packages/frontend-plugin-api/report.api.md

```diff
@@ -982,27 +983,7 @@ export interface ExtensionBoundaryProps {
   routable?: boolean;
 }
 
-// @public (undocumented)
-export type ExtensionDataContainer<UExtensionData extends AnyExtensionDataRef> =
-  Iterable<
-    UExtensionData extends ExtensionDataRef<
-      infer IData,
-      infer IId,
-      infer IConfig
-    >
-      ? IConfig['optional'] extends true
-        ? never
-        : ExtensionDataValue<IData, IId>
-      : never
-  > & {
-    get<TId extends UExtensionData['id']>(
-      ref: ExtensionDataRef<any, TId, any>,
-    ): UExtensionData extends ExtensionDataRef<infer IData, TId, infer IConfig>
-      ? IConfig['optional'] extends true
-        ? IData | undefined
-        : IData
-      : never;
-  };
+export { ExtensionDataContainer };
```

Comment:
This is unfortunate and I was unable to find a quick way for the full type to be reproduced here. Tips welcome.

Reply:
That it including `@internal/frontend` iirc, but maybe that causes trouble

Reply:
Yeah, I think it should work.  One sec.